Utils/AuthenticationKeychainInfo.swift

```diff
@@ -115,6 +115,9 @@ public extension AuthenticationKeychainInfo {
     }
 
     func isLocked() -> Bool {
+        if self.lockOutInterval == nil {
```

Comment:
Nit: Seems a bit clearer if this was a guard statement instead of an if since you're using it to determine a precondition and exit early.
```swift
guard self.locationInterval else { 
    return false 
}
```